### PR TITLE
Dev overhaul sprites

### PR DESCRIPTION
**Pull request by LarsStegman**
I changed the way sprites work.

Please review extensively, it's a pretty big change.

### CODE REVIEW

File: src/main/java/website/frontrow/level/Cell.java [hunk 1]
```diff
@@ -66,7 +66,7 @@ public Sprite getSprite()
         }
     };
 
-    private static SpriteStore ss = new SpriteStore();
+    private static JBubbleBobbleSprites ss = new JBubbleBobbleSprites();
```
Comment (lkroesen) [inline src/main/java/website/frontrow/level/Cell.java]: Please, do not use abreviations.
<naming

File: src/main/java/website/frontrow/sprite/StaticImageSprite.java [hunk 2]
```diff
@@ -51,7 +51,8 @@ public Sprite slice(int x, int y, int width, int height)
         if(withinImage(x, y) && withinImage(x + width - 1, y + height - 1))
         {
             BufferedImage bi = ImageStore.createTranslucentImage(width, height);
```
Comment (lkroesen) [inline src/main/java/website/frontrow/sprite/StaticImageSprite.java]: Please, refactor this abreviation.
<naming

File: src/main/java/website/frontrow/level/Cell.java (hunk 1 above)
Comment (LarsStegman) [inline src/main/java/website/frontrow/level/Cell.java]: See b0af2b5

File: src/main/java/website/frontrow/sprite/StaticImageSprite.java (hunk 2 above)
Comment (LarsStegman) [inline src/main/java/website/frontrow/sprite/StaticImageSprite.java]: See b0af2b5

**Pull request merged**